### ISSUE TITLE
[BITV] 9.2.1.1/3.2 - The "Details" area - The combobox for selecting "Tags" is not fully usable with the keyboard. (2)

### ISSUE THREAD

**Issue by AndyScherzinger** [nextcloud/server#36980]
It is not possible, among other things, to select or edit a tag from the list. Additionally, the keyboard focus in the list is not noticeable. 

<img width="427" alt="5e91a02de42ea9052604ec3630e2fc8e" src="https://user-images.githubusercontent.com/1315170/222244985-031bed68-a5b4-4c2b-b98c-bfbeac5b848a.png">

<details>
<summary>Details</summary>

https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-6bd94ebc49-v3-n2

</details>

- [ ] 9.2.1.1/10.2 - https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-6bd94ebc49-v10-n2

**Comment by ShGKme** [comment]
Seems to be already fixed by moving to `NcSelect`?
![image](https://github.com/nextcloud/server/assets/25978914/abc6c920-ee81-4725-b019-40cdaeb9b106)

**Comment by AndyScherzinger** [comment]
@ShGKme if keyboard navigation works fine for select/edit/remove/add then the keyboard navigation part seems fine. However looking at your screenshot I expect "the keyboard focus in the list is not noticeable." to be still a valid issue that needs fixing (I would expect the contrast to be too low from an a11y perspective would need to be 1:3 or even 1:4,5). Since @JuliaKirschenheuter is on vacation maybe @jancborchardt or @nimishavijay know the right contrast or which to pick (or add a border?)

**Comment by JuliaKirschenheuter** [comment]
If we would use same border as on parent NcSelect element, like 

![Screenshot from 2023-08-14 14-23-28](https://github.com/nextcloud/server/assets/6078378/30d57576-1650-41dc-93be-0f2a5bafc7a4), 
`border: 2px solid var(--color-border-maxcontrast);` then it is ok for a11y.
@nimishavijay @marcoambrosini would you approve such solution?

**Comment by JuliaKirschenheuter** [comment]
>  if keyboard navigation works fine for select/edit/remove/add

select/edit/remove/add seems to work well besides focus (after selecting a tag NcSelect have to be selected itself)

**Comment by jancborchardt** [comment]
@JuliaKirschenheuter yeah sounds good to me! 2 points:
- [ ] If there is a border, we don't need the additional greyish background?
- [ ] The border-radius of the inner element is currently the same as of the inout field/dropdown, which makes it look strangem inner radius needs to be smaller than of the outer element. (Just use the next smaller variable we have.)

**Comment by JuliaKirschenheuter** [comment]
@jancborchardt thanks for reply!

The next smaller border radius size would be `--border-radius` for 3px if i'm not wrong:

![Screenshot from 2023-08-15 16-13-58](https://github.com/nextcloud/server/assets/6078378/9972665c-4e66-4168-ba33-ca594628c56c)

would it be ok for you?

**Comment by ShGKme** [comment]
`NcSelect` border radius is `10px`, and the margin is `4px`, so it should be `10px - 4px = 6px` or `calc(var(--vs-border-radius) - 4px)`, I think.

![image](https://github.com/nextcloud/server/assets/25978914/8b3bcfe4-d682-4c5a-a959-6c168e38caaf)

**Comment by ShGKme** [comment]
> If there is a border, we don't need the additional greyish background?

@jancborchardt 

IMO, the tag without a background is not so noticeable and kind of mixes with the background of NcSelect.

Maybe we can use light-gray color here to make it contrast but still different from the NcSelect background?

**Comment by JuliaKirschenheuter** [comment]
> Maybe we can use light-gray color here to make it contrast but still different from the NcSelect background?

@jancborchardt would such solution be good design-wise? 

![Screenshot from 2023-08-16 15-18-18](https://github.com/nextcloud/server/assets/6078378/d067204c-80a4-4143-b2fd-5eaa0957c7d0)

**Comment by jancborchardt** [comment]
Gopd points @ShGKme – and yes @JuliaKirschenheuter that looks good! :)

**Comment by JuliaKirschenheuter** [comment]
I think we can close a ticket because NcSelect is usable via keyboard
